### ISSUE TITLE
Cannot set custom theme for DatePicker component

### ISSUE THREAD

**Issue by alexd1971** [dart-lang/angular_components#300]
I tried to set custom theme to DateTimePickerComponent using mixin but I failed. I have set custom color theme to other components such as MaterialInputComponent, MaterialCheckbox etc. But DateTimePickerComponent is not working. My sample code is here: https://gist.github.com/alexd1971/84b868f7d2dd97e0fafae34315bf369d
Any help will be appreciated.

**Comment by TedSander** [comment]
material-calendar-picker-theme only changes Ranges which are named 'default' which is not the default range that is created. So this mixin effectively only changes the scrollbar color at this point.

The workaround for now is also add:
```dart
@include calendar-highlights(
      'material-calendar-picker.calendar', (range: $mat-grey-map));
```

I'll need to dig into usages to see if I can switch the default to change the range selection instead.

**Comment by nshahan** [comment]
An update landed that should fix this issue for you 1bf38912b26ed8b748fd308193044ec023fc06f4.

Please feel free to reopen if needed.
